README.md

```diff
@@ -314,6 +314,9 @@ This section contains common parameters that affect all CORTX components running
 | Name                                                  | Description                                                     | Default Value       |
 | ----------------------------------------------------- | --------------------------------------------------------------- | ------------------- |
 | `common.storage_provisioner_path`                     | TODO       | `/mnt/fs-local-volume` |
+| `common.ssl.secret`                                   | Name of a Secret to be generated by `deploy-cortx-cloud.sh` that contains a self-signed certificate.<br />Note: Either `common.ssl.secret` or `common.ssl.external_secret` must be specified. | `""` |
```

Comment:
I don't think this option is necessary. Why don't we just use the name we already do, which is derived from the release ({release name}-ssl-cert)?